src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -67,4 +66,15 @@
    */
   Iterable<BuildInfo> getLastRevisionEntries(int number, int offset)
                                                           throws IllegalArgumentException;
+
+  /**
+   * Queries the database for a given entry, that has the primary key set
```

Comment:
```suggestion
   * Queries the database for a given entry that has the primary key set.
```

Reply:
As per [Mike's review of this class](https://github.com/googleinterns/step240-2020/pull/23/files/cf0e409e17975c3c3fb565d6df31357cf5b88336#diff-1cd265a944af7f7ed73a07e0f6979528), punctuation should be mandatory only for the summary and `@return` parts.

Reply:
Thanks for the information.